ClientLib/src/main/java/com/o3dr/services/android/lib/gcs/follow/FollowLocationSource.java

```diff
@@ -0,0 +1,56 @@
+package com.o3dr.services.android.lib.gcs.follow;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Location source for Follow
+ */
+public enum FollowLocationSource implements Parcelable {
+    Internal("Device GPS"),
+    External("Client Specified")
+    ;
+
+    final String mLabel;
+
+    FollowLocationSource(String label) {
+        mLabel = label;
+    }
+
+    public String getLabel() { return mLabel; }
+
+    @Override
+    public String toString() {
+        return getLabel();
+    }
+
+    public static FollowLocationSource fromOrdinal(int ordinal) {
```

Comment:
@kellyschrock do you foresee this method to be used? I don't really see the point of it otherwise.